app_dart/lib/src/model/appengine/task.dart

```diff
@@ -132,6 +133,11 @@ class Task extends Model {
   @JsonKey(name: 'Reason')
   String reason;
 
+  /// The build ID of luci build: https://chromium.googlesource.com/infra/luci/luci-go/+/master/buildbucket/proto/build.proto#146
+  @IntProperty(propertyName: 'BuildID')
+  @JsonKey(name: 'BuildID')
+  int buildId;
```

Comment:
> > Only adding build id isn't enough. Currently the "build url" is synthesized on the frontend:
> > https://github.com/flutter/cocoon/blob/a6db0e2603dc9e101dd3e54f3e73a8e0c79533b7/app_flutter/lib/logic/qualified_task.dart#L62-L72
> > 
> > Which is vulnerable to future LUCI builder renames. I would save the pool name, builder name in addition, or even the "build url" as a whole.
> 
> @digiter Those info. is better to be saved in config rather than in each task record. I have created issue: [flutter/flutter#58450](https://github.com/flutter/flutter/issues/58450) to track.
> 
> On the other hand, we already have builder info. in the task entity.

Moving the discussion here. Saving the info in `config` would cause troubles when we rename the LUCI pools or LUCI builders. As new task records and old ones will have different URL prefixes, a single `config` wouldn't handle both cases.

Reply:
What is the chance to have such change? The reason I ask is that `task` entity is kind of complex already. Additionally, if we add these info., all records of these property will be same until any change happens. If there is no chance those values change, then it may be good enough to have them in config.

I am okay to save those info., but just want to make sure it makes sense.
/cc @godofredoc

Reply:
I have been thinking about renaming the builders to be more machine friendly and we will probably add devicelab builders in the future.

Reply:
In the future we'll run tasks from different places, e.g. chromium, chrome

Reply:
@digiter Added both `builderName` and `poolName`.